app/feature/feature-odyssey/src/main/kotlin/com/hedvig/android/feature/odyssey/step/singleitemcheckout/SingleItemCheckoutDestination.kt

```diff
@@ -148,11 +283,13 @@ private fun SingleItemCheckoutScreen(
     )
     Spacer(Modifier.height(16.dp))
     Spacer(Modifier.weight(1f))
-    VectorInfoCard(stringResource(R.string.CLAIMS_CHECKOUT_NOTICE), sideSpacingModifier)
+    if (uiState.compensation is ClaimFlowDestination.Compensation.Known.ValueCompensation) {
+      VectorInfoCard(stringResource(R.string.CLAIMS_CHECKOUT_NOTICE), sideSpacingModifier)
+    }
     Spacer(Modifier.height(16.dp))
```

Comment:
If the vector card does not show here, then we got a 16 height space, then the 1f weight then a 16 height space again. Making this button be 32 away from whatever is above, while if the card is there, the card is 16 away from both. Is this expected here?